src/components/FeaturesDemo/index.tsx

```diff
@@ -0,0 +1,726 @@
+                consoleCleanupRef,
+            );
+
+            await pause(PAUSE_SM, signal);
+        },
+        async function closeEditorAndOpenReport() {
+            store.setWindow(1, { pageData: { type: PageTypes.Blank } }, signal);
+
+            await pause(PAUSE_SM, signal);
+
+            store.setHeading(getById(HEADINGS, HeadingId.HtmlReport).data, signal);
+
+            store.popWindow(signal);
+
+            store.pushWindow(
+                {
+                    key: "browser-with-report",
+                    type: ComponentTypes.WebBrowser,
+                    pageData: {
+                        type: PageTypes.Report,
+                        props: {
+                            tests: TESTS_SAMPLE_DATA,
+                        },
+                    },
+                    props: {
+                        url: "localhost:3000",
+                        isPageLoading: false,
+                    },
+                },
+                signal,
+            );
+
+            await pause(PAUSE_LG, signal);
+        },
+        async function selectTestToRun() {
+            store.setHeading(getById(HEADINGS, HeadingId.GuiMode).data, signal);
+            await pause(1000, signal);
+
+            // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+            const testToRunCheckbox = reportBrowserPageRef.current
+                ?.getUnsafePageRef()
+                ?.current?.querySelectorAll("input[type=checkbox]")[1] as HTMLInputElement;
+
+            await reportCursorRef.current?.show();
+            await reportCursorRef.current?.setPosition(
+                testToRunCheckbox.offsetLeft + 1 - 6,
+                testToRunCheckbox.offsetTop + 40 - 6,
+            );
+
+            void reportCursorRef.current?.click();
+
+            await pause(150, signal);
+
+            const newTests = _.cloneDeep(TESTS_SAMPLE_DATA);
+            newTests[0].state = TestState.Indeterminate;
+            newTests[0].children[0].state = TestState.Checked;
+            newTests[0].children[0].children[0].state = TestState.Checked;
+            store.setWindow(
+                1,
+                {
+                    pageData: {
+                        props: {
+                            tests: newTests,
+                        },
+                    },
+                },
+                signal,
+            );
+
+            await pause(PAUSE_SM, signal);
+        },
+        async function clickRunButton() {
+            // eslint-disable-next-line @typescript-eslint/non-nullable-type-assertion-style
+            const runButton = reportBrowserPageRef.current
+                ?.getUnsafePageRef()
+                ?.current?.querySelector('button[data-role-id="run-tests"]') as HTMLButtonElement;
+
+            await reportCursorRef.current?.setPosition(
+                runButton.offsetLeft + 25 - 6,
+                runButton.offsetTop + 1 + 40,
```

Comment:
What these magic values (25, 6, 1, 40) mean?

Reply:
40 is a constant, 25 is just a small offset — left a comment there.